2-mandatory/1-writers.js

```diff
@@ -50,7 +50,7 @@ let writers = [
     occupation: "writer",
```

Comment:
This is again, written by the CYF people, but I again always encourage you wherever possible to change `let` to `const` if you never reassign a variable :) This is nothing wrong, but just good practice!